app.py

```diff
@@ -1639,7 +1658,12 @@ def tryit(level, program_id):
     adventures = load_adventures_for_level(level)
     load_customized_adventures(level, customizations, adventures)
     load_saved_programs(level, adventures, loaded_program)
-    adventures_for_index = load_all_adventures_for_index(customizations)
+    adventures_for_index = load_all_adventures_for_index(customizations, user_programs)
+
+    if current_user()['username']:
+        user_programs = g_db().all_last_programs_for_user(current_user()['username'])
+        print(user_programs)
+        print(adventures_for_index)
```

Comment:
the `print` statements are probably for testing?

Reply:
For sure yes, thanks for pointing it out